lobsterpy/featurize/test/test_core.py

```diff
@@ -486,27 +486,27 @@ def test_lobsterpy_featurize_exception(self):
 
         with self.assertRaises(Exception) as err:
             self.featurize_CsH_cation_anion = FeaturizeLobsterpy(
-                path_to_lobster_calc=TestDir / "TestData/CsH/", bonds="cation-anion"
+                path_to_lobster_calc=TestDir / "TestData/CsH/", bonds="cation_anion"
             )
 
             _ = self.featurize_CsH_cation_anion.get_df()
 
         self.assertEqual(
             err.exception.__str__(),
-            "No cation-anion bonds detected for CsH structure. "
+            "No cation_anion bonds detected for CsH structure. "
             "Please switch to ´all´ bonds mode",
         )
 
         with self.assertRaises(Exception) as err:
             self.featurize_C_cation_anion = FeaturizeLobsterpy(
-                path_to_lobster_calc=TestDir / "TestData/C/", bonds="cation-anion"
+                path_to_lobster_calc=TestDir / "TestData/C/", bonds="cation_anion"
             )
 
             _ = self.featurize_C_cation_anion.get_df()
 
         self.assertEqual(
             err.exception.__str__(),
-            "No cation-anion bonds detected for C structure. "
+            "No cation_anion bonds detected for C structure. "
```

Comment:
Doen't make much sense to write it in this way. I think the changes are not completely thought through yet.